node/10/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM node:10
+
+ENV PHANTOMJS_VERSION 2.1.1
```

Comment:
Do we still need PhantomJS? The last projects that were using it were related to AdCast, like Dashboard UI:
https://github.com/PaeDae/dashboard-ui/blob/develop/package.json#L46

Give that it's a headless browser that's generally used for testing, and since we usually test on a dedicated CI service like Codeship, I don't see a reason for us to have it here. Do you?

Reply:
Here is the discussion Leon and I had on slack.   Please if engineers feel strongly in any which way and understand the issues that can exist when removing items from build packs then let's discuss applying this change.

> Brandon Clark 4:19 PM
oh my it has wand thats huge
If no one minds I would like to remove imagemagick libmagickwand-dev phantomjs bower gulp
Leon Waldman:leonidas: 4:23 PM
Maaaate...
I think this is used in several, undocumented, weird, legacy stuff
Leon Waldman:leonidas: 4:24 PM
yeah... but keep in mind that devs may be counting on it
Brandon Clark 4:24 PM
then maybe in their docker file
that way images stay small as possible
just saying
Leon Waldman:leonidas: 4:25 PM
Thing is... this images are used by the buildpacks
It's assumed that when a dev uses a buildpack  he doesn't know/care about what is docker
(or kubernetes)
because that it's better to leave all this deps in there or we will start to get bug/reports of devs saying that they can't use imagemagick with Node 8
(but can with Node 6... and that this is preposterous)
Brandon Clark 4:28 PM
okay.  Just these aren’t best practice for modern node projects.  I understand you point of view but also see image sizes as an issue for engineers in some services.  Good food for thought but I will keep in the next version.

Reply:
Also should add I would love us to move to `slim` images also.

Reply:
@waldman, we've never used PhantomJS in production; only for testing. One reason why we might want to have it in production, is if we were to create a web crawler. Until then, I don't think it's necessary. My vote is to keep these images as small as possible.